src/porepy/compositional/compositional_mixins.py

```diff
@@ -1171,42 +1211,30 @@ def assign_thermodynamic_properties_to_mixture(self) -> None:
         - :meth:`specific_enthalpy_of_phase` to
           :attr:`~porepy.compositional.base.Phase.specific_enthalpy`
         - :meth:`viscosity_of_phase` to :attr:`~porepy.compositional.base.Phase.viscosity`
-        - :meth:`conductivity_of_phase` to
-          :attr:`~porepy.compositional.base.Phase.conductivity`
+        - :meth:`thermal_conductivity_of_phase` to
+          :attr:`~porepy.compositional.base.Phase.thermal_conductivity`
         - :meth:`fugacity_coefficient` to
           :attr:`~porepy.compositional.base.Phase.fugacity_coefficient_of`
           for each component in respective phase.
           This is only done for mixtures with a defined equilibrium type
 
-        The :attr:`fluid_mixture` gets following properties assigned:
-
-        - :meth:`fluid_density` to :attr:`~porepy.compositional.base.FluidMixture.density`
-        - :meth:`fluid_specific_volume` to
-          :attr:`~porepy.compositional.base.FluidMixture.specific_volume`
-        - :meth:`fluid_specific_enthalpy` to
-          :attr:`~porepy.compositional.base.FluidMixture.specific_enthalpy`
-
-        Customization is possible in respective methods by inheritance.
+        Customization is possible in respective methods by mixing-in.
 
         """
-        assert hasattr(self, "fluid_mixture"), "Mixture not set."
+        assert hasattr(self, "fluid"), "Mixture not set."
 
-        for phase in self.fluid_mixture.phases:
+        for phase in self.fluid.phases:
             phase.density = self.density_of_phase(phase)
             phase.specific_volume = self.specific_volume_of_phase(phase)
             phase.specific_enthalpy = self.specific_enthalpy_of_phase(phase)
             phase.viscosity = self.viscosity_of_phase(phase)
-            phase.conductivity = self.conductivity_of_phase(phase)
+            phase.thermal_conductivity = self.thermal_conductivity_of_phase(phase)
             phase.fugacity_coefficient_of = {}
             for comp in phase:
                 phase.fugacity_coefficient_of[comp] = self.fugacity_coefficient(
                     comp, phase
                 )
 
-        self.fluid_mixture.density = self.fluid_density
-        self.fluid_mixture.specific_volume = self.fluid_specific_volume
-        self.fluid_mixture.specific_enthalpy = self.fluid_specific_enthalpy
-
     def dependencies_of_phase_properties(
```

Comment:
This is a bit tricky.

I hade to change the default behavior of this, because of the mechanics-only models, who inherit from SolutionStrategy.
Problems include the missing of the classical variable mixins like pressure and temperature.

The default return value is now an empty list, leading to fluid properties which raise an error if they were not overridden by some constitutive law.
It makes sense, but may be hard to digest for people digging deeper into the code.

Reply:
Improved documentation, here and at the default implementation of fluid properties in this class.

Reply:
Can be resolved from my side